lms/djangoapps/courseware/module_render.py

```diff
@@ -41,7 +41,7 @@
 
 from common.djangoapps import static_replace
 from common.djangoapps.xblock_django.constants import ATTR_KEY_USER_ID
-from capa.xqueue_interface import XQueueService
+from capa.xqueue_interface import XQueueService  # lint-amnesty, pylint: disable=wrong-import-order
```

Comment:
Moving this import to top causes the tests to fail so suppressed the warning instead.